Automattic/wp-calypso
# Signup: Add Sign In With Apple button

This pull request is a follow-up of https://github.com/Automattic/wp-calypso/pull/35413 which adds a `Sign In With Apple` button on the `User` step of the `Signup` page:

##### WordPress.com

Before | After
------ | -----
![screenshot](https://user-images.githubusercontent.com/594356/63351644-e6066900-c35f-11e9-8cbd-47328244fbcc.png) | ![screenshot](https://user-images.githubusercontent.com/594356/63351670-f3235800-c35f-11e9-915b-2497fbad8b36.png)

##### Crowdsignal

Before | After
------ | -----
![screenshot](https://user-images.githubusercontent.com/594356/63351764-22d26000-c360-11e9-901c-c3ab1f249b8a.png) | ![screenshot](https://user-images.githubusercontent.com/594356/63351798-3251a900-c360-11e9-8495-e05249585e17.png)

##### Jetpack

Before | After
------ | -----
![screenshot](https://user-images.githubusercontent.com/594356/63351852-4c8b8700-c360-11e9-8bc2-71ee218a7511.png) | ![screenshot](https://user-images.githubusercontent.com/594356/63351903-64630b00-c360-11e9-9968-b5eb3089c5b8.png)

Note WooCommerce, Akismet and other OAuth clients don't currently display social login buttons, and as such shouldn't have changed. The width of the social login buttons was updated according to the mockups provided.

#### Testing instructions

Do not hesitate to check how the `Signup` page looks when the width of the browser changes:

1. Run `git checkout add/siwa-to-signup` and start your server, or open a [live branch](https://calypso.live/?branch=add/siwa-to-signup)

##### WordPress.com

2. Open the [`Signup` page](http://calypso.localhost:3000/start/user) in an incognito window
3. Assert that the page looks like the screenshot above

##### Crowdsignal

2. Open the Crowdsignal [`Home` page](https://crowdsignal.com/) in an incognito window
3. Click the `CREATE A SURVEY` button
4. Replace https://wordpress.com with http://calypso.localhost:3000 in the address bar, and reload
5. Assert that the page looks like the screenshot above

##### Jetpack

2. [Create](https://jurassic.ninja/create/) a Jetpack site in an incognito window
3. Click the `Set up Jetpack` green button in the `wp-admin` panel of this site
4. Replace https://wordpress.com with http://calypso.localhost:3000 in the address bar, and reload
5. Click the `Create a new account` link in the footer
6. Replace https://wordpress.com with http://calypso.localhost:3000 in the address bar, and reload
7. Assert that the page looks like the screenshot above

##### Other OAuth apps

2. Open the WooCommerce [`Home` page](https://woocommerce.com/) in an incognito window
3. Click the `GET STARTED` link in the header
4. Replace https://wordpress.com with http://calypso.localhost:3000 in the address bar, and reload
5. Assert that no social button is displayed
